### PR TITLE
the implement of repair the error replications of the volume

### PR DESCRIPTION
**Pull request by stlpmo-jn**
based on the gRPC function VolumeServer.ReplicateVolume()，this future implementing the function of repairing a error replication of the volume

### CODE REVIEW

**Comment by chrislusf** [comment]
Good job! Thanks!
* Please move changes related to weed/topology/replication_health_checker.go to a different PR.
* Fix build issues.

btw: for health check, I am thinking to keep master/topology logic simple, but use shell commands to apply different administrative operations.

File: weed/server/volume_grpc_replicate.go [hunk 1]
```diff
@@ -84,11 +93,35 @@ func (vs *VolumeServer) ReplicateVolume(ctx context.Context, req *volume_server_
...
+	if originFileInf.DatFileSize != uint64(stat.Size()) {
+		return fmt.Errorf("the dat file size [%v] is not same as origin file size [%v]",
+			stat.Size(), originFileInf.DatFileSize)
+	}
+	return nil
 }
 
 func writeToFile(client volume_server_pb.VolumeServer_CopyFileClient, fileName string) error {
-	println("writing to ", fileName)
+	glog.V(4).Infof("writing to ", fileName)
```
Comment (chrislusf) [inline weed/server/volume_grpc_replicate.go]: weed/server/volume_grpc_replicate.go:124: Verbose.Infof call has arguments but no formatting directives

File: weed/topology/replication_health_checker.go [hunk 2]
```diff
@@ -0,0 +1,297 @@
+package topology
```
Comment (chrislusf) [inline weed/topology/replication_health_checker.go]: remove this file from this PR?

Comment (ghost) [reply weed/topology/replication_health_checker.go]: I am so sorry，this file is similar to the command_volume_fix_replication, so i has removed it

Comment (ghost) [reply weed/topology/replication_health_checker.go]: i will reorganize the codes that about the volume replication health checking , and try to make it to shell commands.

**Pull request merged**